src/components/CalendarEventsDialog.vue

```diff
@@ -85,7 +85,7 @@ const calendarOptions = computed<CalendarOption[]>(() => groupwareStore.writeabl
 })))
 const canScheduleMeeting = computed(() => {
 	return hasTalkFeature(props.token, 'schedule-meeting') && store.getters.isModerator && calendarOptions.value.length !== 0
-		&& conversation.value.type !== CONVERSATION.TYPE.ONE_TO_ONE_FORMER
+		&& conversation.value?.type !== CONVERSATION.TYPE.ONE_TO_ONE_FORMER
```

Comment:
Should we backport it as a bug fix, or it is never `undefined` and there is no problem?